src/main/java/creativei/entity/Reminder.java

```diff
@@ -1,18 +1,31 @@
 package creativei.entity;
 
 
+import creativei.vo.ReminderVo;
+import util.LocalizationUtil;
+
 import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import java.text.ParseException;
 import java.util.Date;
 
 @Entity
 public class Reminder extends BaseEntity {
     private String title;
     private String description;
     private Date reminderTime;
+    @ManyToOne
     private Inquiry inquiry;
```

Comment:
Inquiry should be optional per inquiry

Reply:
sir i don't get it. should i have to change the relationship?

Reply:
Make sure this is an optional field

Reply:
It is an optional field.